src/pages/NewChatPage.js

```diff
@@ -65,23 +80,39 @@ class NewChatPage extends Component {
             props.reports,
             props.personalDetails,
             props.betas,
+            '',
+            [],
+            this.props.groupChat ? EXCLUDED_GROUP_EMAILS : [],
```

Comment:
NAB but any reason we shouldn't just pass `EXCLUDED_GROUP_EMAILS` for both groups and single chats?